content/desktop/release-notes.md

```diff
@@ -21,6 +21,55 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-08-29" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.34.0" build_path="/165256/" >}}
+
+### New
+
+- [Host networking](/engine/network/drivers/host.md#docker-desktop) support on Docker Desktop is now generally available.
+- If you authenticate via the CLI without Docker Desktop installed, you can now authenticate through a browser-based flow, removing the need for manual PAT generation.
+- Windows now supports automatic reclamation of disk space in Docker Desktop for WSL2.
```

Comment:
```suggestion
- Windows now supports automatic reclamation of disk space in Docker Desktop for WSL2 installations [using a managed virtual hard disk](/desktop/wsl/best_practices.md). 
```

Because the feature will only work for ~50% of user base (those using the managed VHDX), I am afraid that if we are not specific the highlight will create some confusion for those still using the `docker-desktop-data` distribution.